Source/Libraries/PluginHost/PluginHost.csproj

```diff
@@ -34,7 +34,7 @@
     <Reference Include="System" />
     <Reference Include="System.Configuration" />
     <Reference Include="System.Core" />
-    <Reference Include="System.IO.Compression, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL" />
```

Comment:
This is a change I had to make to remove a build warning, but I'm not sure of its broader implications.

Reply:
I don't think this should have even been referenced in there in the first place, but I'll double-check